projects/hitl/software/tests/conftest.py

```diff
@@ -1,8 +1,35 @@
 import pytest
-from hitl.roadkillharness import RoadkillHarness
+import logging
+from projects.hitl.software.hitl.hitl import Hitl
+from projects.hitl.software.hitl.utils import build_pin, get_logging_config
+from projects.hitl.software.hitl.iocontroller import PinMode, PinType
+
+
+@pytest.fixture(scope="session")
+def hitl(bspd, dbc):
+    hitl = Hitl(bspd, dbc)
+    yield hitl
+
+    hitl.close()
+
+
+@pytest.fixture
+def dbc():
+    return "tests/mkv.dbc"
```

Comment:
Is this really the correct path? Shouldn't it be `vehicle/mkv/mkv.dbc`?

Reply:
Yep, thanks. Changed it in the CAN unit tests but didn't make it call on the DBC fixture so I missed changing this one.